jib-gradle-plugin/CHANGELOG.md

```diff
@@ -10,8 +10,9 @@ All notable changes to this project will be documented in this file.
 ### Changed
 
 - Removed deprecated `jib.jvmFlags`, `jib.mainClass`, `jib.args`, and `jib.format` in favor of the equivalents under `jib.container` ([#461](https://github.com/GoogleContainerTools/jib/issues/461))
-- `jibExportDockerContext` generates different directory layout and `Dockerfile`.
+- `jibExportDockerContext` generates different directory layout and `Dockerfile`
```

Comment:
I know it's not related to this PR, but what does this refer to again? Maybe we should be more specific.

Reply:
Makes sense. I'll put up another PR to make it more specific.